internal/plugins/helm/v1/api.go

```diff
@@ -38,10 +38,7 @@ const (
 	helmChartRepoFlag    = "helm-chart-repo"
 	helmChartVersionFlag = "helm-chart-version"
 
-	defaultCrdVersion       = "v1"
-	defaultHelmChart        = ""
-	defaultHelmChartRepo    = ""
-	defaultHelmChartVersion = ""
```

Comment:
These default values for flags were extracted to consts so that they could be changed easily, as they are found in two different files (in create api subcommand flag binding and in init subcommand check for non-default values when a create api call needs to be run afterwards).